src/app/fyle/split-expense/split-expense.page.ts

```
@@ -814,7 +814,11 @@ export class SplitExpensePage implements OnDestroy {
       this.showErrorBlock = false;
       if (this.amount && parseFloat(this.amount.toFixed(3)) !== this.totalSplitAmount) {
         this.showErrorBlock = true;
-        this.errorMessage = 'Split amount cannot be more than ' + this.amount + '.';
+        if (this.totalSplitAmount < parseFloat(this.amount.toFixed(3))) {
+          this.errorMessage = 'Split amount cannot be less than ' + this.amount + '.';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)